src/types/webpack.ts

```diff
@@ -14,10 +14,12 @@ export interface RegexReplacement {
 export type PlaintextReplacer = (source: string) => string;
 export interface PlaintextPatch {
   find?: string | RegExp;
+  check?: (source: string) => boolean;
```

Comment:
`source` can be optional here, since you can also use `check` for checking if a certain setting is enabled for example. If I'm correct `tsc` will complain about you not using `source` where it is required if you do not provide it.

Reply:
Noted.